exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -376,6 +413,19 @@ table.sortable thead .sorting_desc { background-image: url("/static/img/black-de
       </div>
       <div id="operator-overview" class="panel-collapse collapse">
         <div class="panel-body">
+          <!-- TODO display:none -->
```

Comment:
Why we need todo here and below?

Reply:
Ah.. I needed to hide this by default. For now, it is shown by default, while the toggleWarnings hides it if there is no warning. Will fix this.